File: backend/storage-service/src/main/java/com/example/storageservice/kafka/dto/ProfileDto.java
package com.example.storageservice.kafka.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class ProfileDto {

    private Long id;
    private Long memberId;
    private ProfileType profileType;
    private String value;
    private String createdDate;
    private String updatedDate;
}

File: backend/storage-service/src/main/kotlin/com/example/storageservice/kafka/dto/ProfileDto.kt
package com.example.storageservice.kafka.dto

data class ProfileDto(
    var id: Long? = null,
    var memberId: Long? = null,
    var profileType: ProfileType? = null,
    var value: String? = null,
    var createdDate: String? = null,
    var updatedDate: String? = null,
)
